roadmap/EF2023.md

```diff
@@ -0,0 +1,142 @@
+
+### Hardforks & EIPs
+
+Implementing hardforks and providing early-on access to the added or changed functionality in development tools takes a reoccurring share in our work. We also do selected implementations of dedicated EIPs not yet "hardfork-settled" where it makes sense and help on improving EIP specifications and tests.
+
+### Research, Client & Testnets
+
+The JavaScript team takes part in Ethereum protocol-related research especially in areas where possible future applications in a TypeScript/JavaScript/Web Browser context are on the horizon, going "stateless" is a good example here since it is desirable to have lightweight browser access to the Ethereum network. We use our EthereumJS [client implementation](https://github.com/ethereumjs/ethereumjs-monorepo/tree/master/packages/client) here as a research leveraging tool and for setting up fast-iterating functional testnets for early-on EIP testing in a multi-client context.
+
+### Portal Network
+
+The [Portal Network](https://github.com/ethereum/portal-network-specs) is a new peer-to-peer network topology for the Ethereum network (like devp2p) being built to allow for lightweight Ethereum (mainnet) protocol access and expose a JSON-RPC like interface along. The effort is led by [Piper Merriam](https://github.com/pipermerriam) and progress has been substantially advanced with specs laid out on a detailed level and two client implementations (Trin [Rust] and Fluffy [Nim]) underway.
+
+The EF JavaScript team has built a [prototype](https://github.com/acolytec3/portalnetwork) of this in 2021 to test its feasibility. In 2022 this became a steady project with a dedicated team and a full-featured JS-based Portal Network client is now developed within the realm of the [Ultralight](https://github.com/ethereumjs/ultralight) monorepo. 2023 will be the year where the Portal Network will get closer to a production state and hopefully early-on integrations with dAppps and/or Wallet providers can take place to test a wider roll-out of the new network topology.
+
+### Ethers.js
+
+The [Ethers.js](https://github.com/ethers-io/ethers.js) library is one of the most popular connector and wallet libraries used to and interact with the Ethereum network. Primarily maintained by [Richard Moore](https://github.com/ricmoo) the library is downloaded by far over 1 million users and developers each month. To further develop and maintain the library is therefore crucial to support the large community which has grown around the library over the years.
```

Comment:
```suggestion
The [Ethers.js](https://github.com/ethers-io/ethers.js) library is one of the most popular tooling libraries used to and interact with the Ethereum network. Primarily maintained by [Richard Moore](https://github.com/ricmoo) the library is downloaded by far over 1 million users and developers each month. To further develop and maintain the library is therefore crucial to support the large community which has grown around the library over the years.
```
Since Ethers does so many things, I'd call it a "tooling library" as it does more that connectivity/wallet imo (e.g. utils for parsing/encoding/decoding etc.)

Reply:
👍